### PR TITLE
Added json file for new gateway

### PR DESCRIPTION
**Pull request by leo-lizzi**

### CODE REVIEW

File: AA555A0000000000.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "AA555A0000000000",
```
Comment (gonzalocasas) [inline AA555A0000000000.json]: This doesn't look like a real EUI.

Comment (leo-lizzi) [reply AA555A0000000000.json]: You are right. I took the wrong number. How can I modify the file?

Comment (leo-lizzi) [reply AA555A0000000000.json]: I uploaded the correct file.

**Comment by gonzalocasas** [comment]
@leo-lizzi could you please delete the old file completely? Thanks!

**Pull request merged**